otter/controller.py

```diff
@@ -98,6 +98,16 @@ def __init__(self, tenant_id, group_id, operation, extra=None):
             fmt.format(t=tenant_id, g=group_id, o=operation))
 
 
+class TenantSuspendedError(Exception):
```

Comment:
Why? Whats wrong with regexing the string? I think its a perfectly valid solution over storing the attribute :P (just kidding)